packages/block-library/src/file/save.js

```diff
@@ -11,16 +12,42 @@ export default function save( { attributes } ) {
 		textLinkTarget,
 		showDownloadButton,
 		downloadButtonText,
+		displayPreview,
+		previewHeight,
 	} = attributes;
 
+	const pdfEmbedLabel = RichText.isEmpty( fileName )
```

Comment:
I'm a bit confused with this check. You should add the `aria-label` only if the filename exists and you should apply the `pdf` label with a check like you are doing in `edit.js` file (`href.endsWith( '.pdf' ) `), no?

Reply:
I went with adding a default `aria-label`, since I figured it's better to give the embed a generic label, rather than no label at all.

I skipped the `href.endsWith( '.pdf' )` check, since the label is only used behind a `displayPreview` check. Technically it means that `pdfEmbedLabel` will be set for non-PDF files, but the value won't be used in the save result.